src/NuGet.Core/NuGet.ProjectModel/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,5 @@
+NuGet.ProjectModel.ProjectRestoreMetadataFrameworkInfo.TargetAlias.get -> string
+NuGet.ProjectModel.ProjectRestoreMetadataFrameworkInfo.TargetAlias.set -> void
+NuGet.ProjectModel.TargetFrameworkInformation.TargetAlias.get -> string
+NuGet.ProjectModel.TargetFrameworkInformation.TargetAlias.set -> void
+static NuGet.ProjectModel.PackageSpecExtensions.GetTargetFramework(this NuGet.ProjectModel.PackageSpec project, string targetAlias) -> NuGet.ProjectModel.TargetFrameworkInformation
```

Comment:
It's used in MSBuildRestoreUtility. 

I can just define an internal extension if you want me to. I don't necessarily know if we'll need it outside of commands...probably later if we try to allow duplicates.

Reply:
It's not awful, so I won't block the PR on it, but from an API design point of view, I don't like it. If it exists because expecting callers to use `packageSpec.TargetFrameworks.SingleOrDefault(...)` is considered too hard, then we should probably consider a `packageSpec.TargetFrameworks[string]` indexer, rather than this helper extension.  Plus my comment on the implementation of the method itself, where its behaviour is not at all what I personally would expect (return value when requested key does not exist)

Reply:
removed.